openvpn-install.sh

```diff
@@ -572,6 +588,12 @@ ifconfig-pool-persist ipp.txt" >> /etc/openvpn/server.conf
 		echo 'push "dhcp-option DNS 176.103.130.130"' >> /etc/openvpn/server.conf
 		echo 'push "dhcp-option DNS 176.103.130.131"' >> /etc/openvpn/server.conf
 		;;
+		10) # Custom DNS
+		echo """push "dhcp-option DNS $DNS1"""" >> /etc/openvpn/server.conf
```

Comment:
Why all that quotes, as there aren't any special characters that need escaping. Can't just be `'push "dhcp-option DNS $DNS1"'`?

Reply:
The inside quotes are mandatory to paste them on file. With single quotes I cannot pass variables.